taurunner/modules/check_specs.py

```diff
@@ -0,0 +1,8 @@
+def check_specs(TR_specs):
+    
+    if TR_specs['nevents'] <= 0::
```

Comment:
Double colon syntax error is breaking the checks so we can't see if this PR is breaking the tests